backend/app/api/endpoints/projects.py

```
@@ -176,6 +177,30 @@ def create_project_conversation_endpoint(
         )
 
 
+@router.post(
+    "/{project_id}/archive-chats",
+    response_model=TaskArchiveBatchResponse,
+)
+def archive_project_chats_endpoint(
+    project_id: int = Path(..., description="Project ID"),
+    current_user: User = Depends(get_current_user),
```

Ruff (0.15.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)